mms/export_model.py

```diff
@@ -240,7 +240,7 @@ def find_unique(files, suffix):
 
 
 def validate_epoch_number(params_file):
-    if re.match(r'^\w+-\d+\.params$', params_file) is None:
+    if re.match(r'^.+-\d+\.params$', params_file) is None:
```

Comment:
thanks for submitting this!

matching any character seems to broad -- that could include spaces or other non-url compatible characters. can you change it to `r'^[\w\-]+-\d+\.params$'` ?

Reply:
Thanks for your comment.

How about using `r'^[\w\-\.]+-\d+\.params$'` so that `squeezenet_v1.1-0000.params` shown in the export example can also be used. (allow hyphens and dots.)

Reply:
yes, that would be perfect.

Reply:
Thanks.
Correct the code and commit it.